smarts/core/sensors.py

```diff
@@ -799,7 +796,7 @@ def __init__(
         self,
         vehicle,
         bullet_client,
-        sensor_params: SensorParams = None,
+        sensor_params: Optional[SensorParams] = None,
         lidar_offset=(0, 0, 1),
```

Comment:
Why are there no type defs for other three attributes?

Reply:
Here, I had simply fixed an erroneous type caught by pytype.